fern/pages/sdks/getting-started/preview-your-sdk-locally.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Preview your SDK
+description: Use Fern's CLI tool to preview your SDK locally. 
```

Comment:
can we also add a subtitle here that shows up in the docs
<img width="387" alt="Screenshot 2024-10-02 at 1 00 54 PM" src="https://github.com/user-attachments/assets/af44db68-cf4b-422a-a968-243070a3ae89">


Would make it `Preview your SDK locally by running `fern generate --preview`